pkg/analysis/numericbounds/testdata/src/a/a.go

```diff
@@ -0,0 +1,152 @@
+type InvalidInt64ExceedsJSBothBounds struct {
+	// +kubebuilder:validation:Minimum=-9007199254740992
+	// +kubebuilder:validation:Maximum=9007199254740992
+	UnsafeBoth int64 // want "field UnsafeBoth of type int64 has bounds \\[-9007199254740992, 9007199254740992\\] that exceed safe integer range \\[-9007199254740991, 9007199254740991\\]\\. Consider using a string type to avoid precision loss in JavaScript clients"
+}
+
+// IgnoredStringField should not be checked
+type IgnoredStringField struct {
+	Name string
+}
+
+// IgnoredBoolField should not be checked
+type IgnoredBoolField struct {
+	Enabled bool
+}
+
+// IgnoredFloat64Field should not be checked
+type IgnoredFloat64Field struct {
+	Value float64
+}
+
+// MixedFields has both valid and invalid fields
+type MixedFields struct {
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=100
+	ValidCount int32
+
+	InvalidCount int32 // want "field InvalidCount of type int32 should have minimum and maximum bounds validation markers"
+
+	Name string
+
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=1000
+	ValidValue int64
+
+	InvalidValue int64 // want "field InvalidValue of type int64 should have minimum and maximum bounds validation markers"
+}
+
+// PointerFields with pointers should also be checked
+type PointerFields struct {
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=100
+	ValidPointerWithBounds *int32
+
+	InvalidPointer *int32 // want "field InvalidPointer of type int32 should have minimum and maximum bounds validation markers"
+
+	InvalidPointer64 *int64 // want "field InvalidPointer64 of type int64 should have minimum and maximum bounds validation markers"
+}
+
+// SliceFields with slices should check the element type
+type SliceFields struct {
+	ValidSlice []string
+
+	InvalidSlice []int32 // want "field InvalidSlice of type int32 should have minimum and maximum bounds validation markers"
+
+	InvalidSlice64 []int64 // want "field InvalidSlice64 of type int64 should have minimum and maximum bounds validation markers"
+
+	// +kubebuilder:validation:items:Minimum=0
+	// +kubebuilder:validation:items:Maximum=100
+	ValidSliceWithBounds []int32
+}
+
+// TypeAliasFields with type aliases should be checked
+type Int32Alias int32
+type Int64Alias int64
+
+type TypeAliasFields struct {
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=100
+	ValidAlias Int32Alias
+
+	InvalidAlias Int32Alias // want "field InvalidAlias of type int32 should have minimum and maximum bounds validation markers"
+
+	InvalidAlias64 Int64Alias // want "field InvalidAlias64 of type int64 should have minimum and maximum bounds validation markers"
+}
+
+// PointerSliceFields with pointer slices should also be checked
+type PointerSliceFields struct {
+	InvalidPointerSlice []*int32 // want "field InvalidPointerSlice of type int32 should have minimum and maximum bounds validation markers"
```

Comment:
We generally have the message here as something along the lines of `field InvalidPointerSlice array element type of int32...` Please search for examples of this and be consistent